js/background.js

```diff
@@ -73,6 +74,8 @@
         return accountManager;
     };
 
+    const cancelInitializationMessage = Views.Initialization.setMessage();
```

Comment:
I feel like 'clear' is probably better than 'cancel' here.

Reply:
To clarify, calling this function is not clearing the optimization message—`AppLoadingScreen` does that—but only canceling its appearance if it hasn’t been shown yet. In other words, once **Optimizing application…** appears, it stays until another view overwrites it. This function doesn’t reset the text of the `.message` `div` hence `cancel*` instead of `clear*`.